app.js

```diff
@@ -0,0 +1,21 @@
+import express from 'express';
+import { connectToDb, getDb } from './db.js';
+import userRoutes from './routes/userRoutes.js';
+
+const app = express();
+
+app.use(express.json());
+app.use('/api', userRoutes);
+
+connectToDb((error) => {
+  if (!error) {
+    app.listen(5000, () => {
+      console.log('App listening on port 5000');
+    });
+    getDb();
+  }
+});
+
+app.get('/', (req, res) => {
+  res.json({ msg: 'Welcome!' });
+});
```

Comment:
Unnecessary,  instead you can create a /health endpoint that returns `200 OK` if `mongoose` is connected.